lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/README.md

```diff
@@ -106,7 +106,7 @@ The function has the following additional parameters:
 -   **offsetX**: starting index for `x`.
 -   **offsetOut**: starting index for `out`.
 
-While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying `buffer`, the `offset` parameter supports indexing semantics based on a starting index. For example, to calculate the sum of every other value in `x` starting from the second value
+While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, offset parameters support indexing semantics based on starting index. For example, to calculate the sum of every other value in `x` starting from the second value:
```

Comment:
```suggestion
While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, offset parameters support indexing semantics based on starting indices. For example, to calculate the sum of every other value in `x` starting from the second value:
```

Grammar. You need parallel construction. If you change the subject to plural, you also need to update the dependent clauses accordingly.